controller/setting_controller.go

```diff
@@ -563,6 +567,51 @@ func (sc *SettingController) updatePriorityClass() error {
 	return nil
 }
 
+func (sc *SettingController) updateKubernetesClusterAutoscalerEnabled() error {
+	setting, err := sc.ds.GetSetting(types.SettingNameKubernetesClusterAutoscalerEnabled)
+	if err != nil {
+		return err
+	}
+	newClusterAutoscalerEnabled := setting.Value
+	evictKey := types.KubernetesClusterAutoscalerSafeToEvictKey
+	deploymentList, err := sc.ds.ListDeploymentWithLabels(types.GetBaseLabelsForSystemManagedComponent())
+	if err != nil {
+		return errors.Wrapf(err, "failed to list Longhorn deployments for priority class update")
+	}
+	longhornUI, err := sc.ds.GetDeployment(types.LonghornUIDeploymentName)
```

Comment:
Please use extra new line following where you finish one logic group of statements. This is kind of coding convention as the below example.

```go
    deploymentList, err := sc.ds.ListDeploymentWithLabels(types.GetBaseLabelsForSystemManagedComponent())
	if err != nil {
		return errors.Wrapf(err, "failed to list Longhorn deployments for priority class update")
	}

	longhornUI, err := sc.ds.GetDeployment(types.LonghornUIDeploymentName)
	if err != nil && !apierrors.IsNotFound(err) {
		return errors.Wrapf(err, "failed
```

Reply:
Fixed.